docs/_documentations/mdt-vsc-getting-started.md

```diff
@@ -12,8 +12,7 @@ parent: mdt-vsc-overview
 
 # Installing Codewind for VS Code
 <br/>
-
- Codewind for VS Code enables you to develop your containerized projects from within VS Code. 
+ Codewind for Visual Studio (VS) Code enables you to develop your containerized projects from within VS Code. 
```

Comment:
In other files, I seem to recall the abbreviation being formatting like "Visual Studio Code (VS Code)". Not a huge deal...but another little consistency thing.